### PR TITLE
[FIX] Invite links counting users already joined

### LINKED ISSUE #16573: Invite links uses count even if the user is already in the room.
### Description:

Users can open invite links multiple times, even if they joined the room already. This should not happen, as invite links can have limited uses.

### Steps to reproduce:

1. Go to any room.
2. Click on the users icon then invite users.
3. Copy the link and access it multiple times on the same account.
4. Go to Admin->Invites: Check your invite link, it should have counted every time you entered the room through it.

### Expected behavior:

Count only new users to the room.

### Actual behavior:

It counts every single attempt.

### Server Setup Information:

- Version of Rocket.Chat Server: 3.0.0-develop
- Operating System: macOS Catalina 10.15.2
- Deployment Method: Development
- Number of Running Instances: 1
- DB Replicaset Oplog: True
- NodeJS Version: 13.5.0
- MongoDB Version: 4.0.6

### Client Setup Information

- Desktop App or Browser Version: Chrome Version 80.0.3987.87 (Official Build) (64-bit)
- Operating System: macOS Catalina 10.15.2

### PR DESCRIPTION
**Pull request by ritwizsinha**
Closes #16573 
out

### CODE REVIEW

**Comment by ritwizsinha** [comment]
@gabriellsh please review

File: app/invites/server/functions/useInviteToken.js [hunk 1]
```diff
@@ -18,7 +18,11 @@ export const useInviteToken = (userId, token) => {
 	const user = Users.findOneById(userId);
 	Users.updateInviteToken(user._id, token);
 
-	Invites.increaseUsageById(inviteData._id);
+	const subscription = Subscriptions.findOneByRoomIdAndUserId(room._id, user._id, { fields: { _id: 1 } });
+	const userInRoom = subscription != null;
```
Comment (gabriellsh) [inline app/invites/server/functions/useInviteToken.js]: Please change to !!subscription. Thanks for the help.

File: app/invites/server/functions/useInviteToken.js [hunk 2]
```diff
@@ -18,7 +18,11 @@ export const useInviteToken = (userId, token) => {
...
+	if (!userInRoom) {
```
Comment (gabriellsh) [inline app/invites/server/functions/useInviteToken.js]: you don't need a const for this, just do !subscription. Thanks for the help

Comment (ritwizsinha) [reply app/invites/server/functions/useInviteToken.js]: Made the changes

**Pull request merged**